qiskit_ibm_runtime/session.py

```diff
@@ -193,6 +193,38 @@ def backend(self) -> Optional[str]:
         """
         return self._backend
 
+    def status(self) -> Optional[str]:
+        """Return current session status."""
+        # TODO implement this when API changes are done
```

Comment:
What does this mean? The session status is just the `state` field.

Reply:
Originally I thought we would have to map the state `'active' | 'open' | 'inactive' | 'closed'`, to some status like `in progress, accepting new jobs` - but I believe that's going to be handled on the server side now so I can update this when ready.

Reply:
>but I believe that's going to be handled on the server side now so I can update this when ready.

That's news to me...